core/modules/telemetry/telemetry.admin.inc

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Administrative pages for Telemetry module.
+ */
+
+function telemetry_report() {
+  $telemetry_info = telemetry_get_info();
+  $telemetry_data = telemetry_get_data();
+
+  $module_path = backdrop_get_path('module', 'telemetry');
+  backdrop_add_css($module_path . '/css/telemetry.admin.css');
+  $page = array(
+    '#type' => 'container',
+    '#attributes' => array('class' => array('telemetry-report')),
+  );
+
+  $help = '';
+  $help .= '<p>' . t('Help us improve Backdrop CMS by sending anonymous, untracked information about what features your site uses.') . '</p>';
+  $help .= '<p>' . t('Below is a list of all data collected by the enabled modules on your site. Only projects that are publicly available on BackdropCMS.org will store usage information.') . '</p>';
+  $help .= '<p>' . t('If you prefer to not send usage information, you can disable the Telemetry module from the <a href="!modules">modules page</a>.', array('!modules' => url('admin/modules'))) . '</p>';
+
+  $page['help'] = array(
+    '#type' => 'help',
+    '#markup' => $help
+  );
+
+  $header = array(
+    t('Module'),
+    t('Description'),
+    t('Value'),
+  );
+
+  $rows = array();
+  foreach ($telemetry_info as $project => $project_data) {
+    foreach ($project_data as $key => $info) {
+      $row = array();
+      if (!isset($telemetry_data[$project][$key])) {
+        $value = '<span class="error">Missing value</span>';
+      }
+      else {
+        $value = check_plain($telemetry_data[$project][$key]);
+      }
+      $module_info = system_get_info('module', $info['module']);
+      $project_name = $info['project'] === 'backdrop' ? 'core' : check_plain($info['project']);
+      $row[] = '<div class="telemetry-item-module">' . check_plain($module_info['name']) . '</div><div class="telemetry-item-project">' . $project_name . '</div>';
+      $row[] = '<div class="telemetry-item-label">' . check_plain($info['label']) . '</div><div class="telemetry-item-description">' . check_plain($info['description']) . '</div>';
+      $row[] = array(
+        'data' => $value,
+        'class' => array('telemetry-item-value'),
+      );
+      $rows[] = $row;
+    }
+  }
+
+  $page['items'] = array(
+    '#theme' => 'table',
```

Comment:
Should we make this tableselect, and allow users to choose? Once contrib starts implementing this, could be users may prefer a choice.

Reply:
@docwilmot I think that would be a good idea for a follow-up. For now with only three items showing, we probably are okay.

Reply:
My original implementation actually did have this as tableselect so that you could turn on and off specific keys. However the discussion in the meta issue took the direction of having all telemetry data be all-or-nothing. In truth that made things _much_ simpler, because determining how to add/request approval for new keys as modules update adds a whole new problem. Also the ability to turn off every key was a bit overwhelming. Most other platforms that collect telemetry data seem to be all-or-nothing.